src/TableFunctions/TableFunctionValues.cpp

```diff
@@ -83,7 +82,7 @@ void TableFunctionValues::parseArguments(const ASTPtr & ast_function, ContextPtr
     ASTs & args_func = ast_function->children;
 
     if (args_func.size() != 1)
-        throw Exception("Table function '" + getName() + "' must have arguments", ErrorCodes::LOGICAL_ERROR);
```

Comment:
Second, it's not the only place where exception like this is thrown. Just grep `must have arguments", ErrorCodes::LOGICAL_ERROR` and you will find at least 2 more places. Should we fix other places as well?